src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -74,27 +75,38 @@ internal sealed class EslintBridgeProcess : IEslintBridgeProcess
 
         public Task<int> Start()
         {
-            if (startTask != null && !startTask.Task.IsFaulted)
+            lock (Lock)
```

Comment:
Should prevent spawning multiple processes during first time analysis/after failure.